backend/apps/owasp/templates/search/chapter.html

```diff
@@ -125,7 +125,7 @@ <h4 class="px-1">
             methods: {
                 async getChapters() {
 
-                    const response = await fetch(`/api/v1/owasp/search/chapter?q=${this.searchQuery}`)
+                    const response = await fetch(`/api/v1/owasp/search/chapter?q=${this.searchQuery}&page=1`)
```

Comment:
That's the default one, right?
I think we can avoid the old backend code updates.

Reply:
yes

Reply:
We don't use the new frontend yet. It's in dev only.
My point it that you don't need to change the query as page=1 is the default value on the backend.

Reply:
okk